src/GitVersionCore/Configuration/ConfigurationProvider.cs

```diff
@@ -31,15 +31,15 @@ public static void ApplyDefaultsTo(Config config)
             var configBranches = config.Branches.ToList();
 
             ApplyBranchDefaults(config, GetOrCreateBranchDefaults(config, "master"), defaultTag: string.Empty, defaultPreventIncrement: true);
-            ApplyBranchDefaults(config, GetOrCreateBranchDefaults(config, "release[/-]"), defaultTag: "beta", defaultPreventIncrement: true);
```

Comment:
I guess I'm not sure how the migration should work? Should apply branch defaults look for the old keys and rename them?